src/database/hooks/useLocalDatabaseHook.js

```diff
@@ -36,7 +37,17 @@ const useLocalDatabaseHook = (withMigration = false) => {
     }));
   };
 
-  const {channelInfo, channelList, channelMember, chat, user} = databaseListener;
+  const debouncedRefresh = React.useCallback(
+    _.debounce((key) => refresh(key)),
+    50
+  );
+
+  const refreshWithId = (key, id) => {
+    const keyWithId = `${key}_${id}`;
+    debouncedRefresh(keyWithId);
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `debouncedRefresh` and `refreshWithId` functions is a good practice as it helps in reducing unnecessary refreshes and provides a way to refresh data based on specific IDs. However, ensure that the debounce time (50ms) is appropriate for your use case. If the debounce time is too short, you might not get the desired throttling effect.



<!-- This is an auto-generated comment by OSS CodeRabbit -->